eq-author-api/migrations/upateAnswerDeletedMetaData.js

```diff
@@ -0,0 +1,55 @@
+const { getPages } = require("../schema/resolvers/utils");
+
+module.exports = (questionnaire) => {
+  const ctx = {
+    questionnaire,
+  };
+  const { metadata } = questionnaire;
+  const pages = getPages(ctx);
+  const regexp = /[\w]{8}-[\w]{4}-[\w]{4}-[\w]{4}-[\w]{12}/i;
+
+  pages.map((page) => {
+    if (page?.title?.includes("metadata")) {
+      const splitTitleToArray = page.title.split(/(?<=\/span)/);
+
+      splitTitleToArray.map((titlePart) => {
+        const idExists = regexp.exec(titlePart);
+
+        if (idExists !== null && titlePart.includes("metadata")) {
+          const index = splitTitleToArray.indexOf(titlePart);
+
+          const titleMetadtaId = idExists[0];
```

Comment:
```suggestion
          const titleMetadataId = idExists[0];
```
Dismiss these requested changes if they are not important